game_server_launcher/gameserverhandler.py

```diff
@@ -73,6 +73,7 @@ def run_game_server(game_server_config):
     try:
         working_dir = game_server_config['dir']
         dll_to_inject = game_server_config.get('controller_dll', None)
+        dll_config_path = game_server_config.get('controller_config', None)
```

Comment:
What do you think would be the best default location? I think next to the other configs we have. Or are there other reasons why the default path in Documents is a better choice?

Reply:
I'm putting it with the other configs, but in its own `controller` folder as it's not unlikely for there to be multiple files (e.g. the goty preset I've been working on does this).

I'm also making it a mandatory config parameter since I think it's better behaviour than having it default to my documents if you don't set it.